.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
```

Comment:
can we hookup coverage as well?

Reply:
ah i guess it's still run from buildkite on monorepo... @yihau do we have any opinions on a coverage action for new repos?

Reply:
maybe we can steal the idea from the sdk repo haha
https://github.com/anza-xyz/solana-sdk/blob/5c389a1dc95b4a267aae44f4e43bf32386aab027/.github/workflows/main.yml#L499-L520

however, it requires some environment setup. we can add that later so it wouldn't block the important part!

Reply:
ok yeah let's punt to a follow up. would be good to get coverage enabled and ensure everything is well tested before 1.0.0